lib/eurocalliopelib/utils.py

```
@@ -49,12 +53,99 @@ def convert_country_code(input_country, output="alpha3"):
         return pycountry.countries.lookup(input_country).alpha_3
 
 
-# conversion utils
+def convert_valid_countries(country_codes: list, output: str = "alpha3") -> dict:
+    """
+    Convert a list of country codes / names to a list of uniform ISO coded country
+    codes. If an input item isn't a valid country (e.g. "EU27") then print the code and
```

I'd be careful around skipping exceptions by default, since this is an utility function.

I'd introduce a flag to ignore exceptions, with the default being raising them. This helps avoid issues down the line since exceptions skipping is made explicit in the code calling this.